src/proof/bid_tests.rs

```diff
@@ -154,12 +145,11 @@ mod protocol_tests {
         let latest_consensus_step = 50u64;
 
         // Append the Bid to the tree.
-        tree.push(bid.into());
+        tree.push(bid.into()).unwrap();
```

Comment:
Use `expect` at least, or since the test returns a `Result` checks if you can use `?` or `map_err`.